streams/src/test/java/org/apache/kafka/streams/processor/internals/StandbyTaskTest.java

```diff
@@ -247,23 +231,19 @@ public void shouldOnlyCheckpointStateWithBigAdvanceIfNotEnforced() {
         assertEquals(Collections.singletonMap(partition, 11000L), task.offsetSnapshotSinceLastFlush);
         task.maybeCheckpoint(false);  // this should not checkpoint
         assertEquals(Collections.singletonMap(partition, 11000L), task.offsetSnapshotSinceLastFlush);
-
-        EasyMock.verify(stateManager);
     }
 
     @Test
     public void shouldFlushAndCheckpointStateManagerOnCommit() {
-        EasyMock.expect(stateManager.changelogOffsets()).andStubReturn(Collections.emptyMap());
+        when(stateManager.changelogOffsets()).thenReturn(Collections.emptyMap());
         stateManager.flush();
-        EasyMock.expectLastCall();
+        verify(stateManager, atLeastOnce()).flush();
```

Comment:
You again call a function on the mock and immediately verify it. Maybe you just missed to remove it here, since you added the verification on line 259. However, I do not understand why you use `atLeastOnce()`. It should be called only once according to the pre-migration code. If you remove `stateManager.flush()` on line 239, you will see that it is indeed only called once.